src/pages/HomePage/HomePageComponents/CategoryCard/CategoryCard.css

```diff
@@ -0,0 +1,37 @@
+.categories {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.card-image {
+    width: 100%;
+    height: 300px;
+    object-fit: cover;
+    object-position: bottom;
+}
+
+.card {
+    width: 20rem;
+    margin: 0 1rem;
+}
```

Comment:
prefer using rems for anything above 10 px

Reply:
Sure...
Will implement it 👍️